Containerized Docker Application Lifecycle - 01.1 - Chapter 01 - Article 1

mvelosop opened the pull request:
## Summary

### e-book <--> docs.microsoft.com synchronization
- Chapter 01 - article 1

Related PR: https://github.com/dotnet/docs/pull/9305

mvelosop committed:
Containerized Docker Application Lifecycle - Article 01.1

mairaw commented on docs/standard/containerized-lifecycle-architecture/index.md:
```diff
@@ -5,25 +5,23 @@ author: CESARDELATORRE
 ms.author: wiwagn
 ms.date: 09/22/2017
```
change ms.date?

mairaw commented on docs/standard/containerized-lifecycle-architecture/index.md:
```diff
@@ -5,25 +5,23 @@ author: CESARDELATORRE
 ms.date: 09/22/2017
 ---
-# Introduction to containers and Docker
+# Introduction to Containers and Docker
```
Previous title was the correct capitalization

```suggestion
# Introduction to containers and Docker
```

mairaw commented on docs/standard/containerized-lifecycle-architecture/index.md:
```diff
@@ -5,25 +5,23 @@ author: CESARDELATORRE
 ms.author: wiwagn
```
can we fix the description here for SEO?

mairaw reviewed (changes requested):
Small nits to fix but looks good overall

mvelosop commented:
@mairaw, we had already fixed metadata in PR https://github.com/dotnet/docs/pull/9305, so I just cherry-picked this commit and applied to a fresh branch (and fixed the title casing)
